### ISSUE TITLE
Do not parse nested objects.

### ISSUE THREAD

**Issue by tumdav** [richardgirges/express-fileupload#51]
**Request example**
______
![image](https://user-images.githubusercontent.com/10452309/30734458-4778e3c4-9f8c-11e7-96f4-447321bb8e00.png)

**Expected Result**

```json
{  
   "name":"Yerevan Mall Food Court",
   "layout":{  
      "width":500,
      "height":400
   }
}
```
**Actual Result**
______
```json
{  
   "name":"Yerevan Mall Food Court",
   "layout[height]":500,
   "layout[width]":400
}
```

**Comment by richardgirges** [comment]
Hey thanks for the report @tumdav 

I will try to get to this in the next week or so. If you happen to have the bandwidth to send the PR, it would be greatly appreciated!

**Comment by tumulr** [comment]
@richardgirges I want to help but I am not able to understand. Can you explain me what needs to be done?

**Comment by richardgirges** [comment]
@tumulr you would probably want to parse the req.body property and convert it into nested objects. This will be a breaking change so we'll need to make a minor release update. 

My guess is that there may be a lightweight package out there that we can use for converting the form-based nested structures into actual JSON nested structures - although it would be nice to implement this raw without introducing additional dependencies.

**Comment by richardgirges** [comment]
Resolved by #110
